dispatchr-ruby/spec/controllers/requests_controller_spec.rb

```diff
@@ -24,30 +26,80 @@
 	describe 'POST #create' do
 		context 'when request is successfully created' do
 			it 'returns 200 status' do
-				post :create, {user_id: 1, expiration_date: '0'}
+				create_request
 				expect(response).to have_http_status(200)
 			end
+
 			it 'has the correct user_id and expiration_date' do
-				post :create, {user_id: 1, expiration_date: '0'}
+				create_request
 				request = Request.last
 				expect(request.user_id).to eq(1)
-				expect(request.expiration_date).to eq(nil)
+				expect(request.expiration_date).to eq("0028-11-16T00:00:00.000Z")
```

Comment:
This spec will start failing after the 28th. Why not do a farther away expiration date?